isis/src/control/objs/BundleSolutionInfo/BundleSolutionInfo.cpp

```diff
@@ -1385,8 +1502,16 @@ namespace Isis {
 
       // Removed radiansToMeters argument 9/18/2018 DAC
       QString pointDetailString =
-          bundleControlPoint->formatBundleOutputDetailString(berrorProp,
-                                                           solveRadius);
+          bundleControlPoint->formatBundleOutputDetailString(berrorProp, solveRadius);
+      fpOut << (const char*)pointDetailString.toLatin1().data();
+    }
+
+    for (int i = 0; i < nLidarPoints; i++) {
+      BundleLidarControlPointQsp bundleLidarControlPoint =
+          m_statisticsResults->bundleLidarControlPoints().at(i);
+
+      QString pointDetailString =
+          bundleLidarControlPoint->formatBundleOutputDetailString(berrorProp, solveRadius);
```

Comment:
Same with this?

Reply:
Same as above, this whole for loop will never execute.